### ISSUE TITLE
Global seam leveling

### ISSUE THREAD

**Issue by smathermather** [dronemapper-io/NodeMICMAC#17]
When processing the [OpenDroneMap Aukerman example](https://github.com/OpenDroneMap/odm_data_aukerman) we get some issue with global seam leveling (ht this unrelated issue: https://github.com/dronemapper-io/NodeMICMAC/issues/15).

ODM version:
![image](https://user-images.githubusercontent.com/1174901/58280331-56f0b180-7d6e-11e9-9974-13ab07d350aa.png)

NodeMICMAC version:
![image](https://user-images.githubusercontent.com/1174901/58280347-5c4dfc00-7d6e-11e9-842f-b90d1d206e7e.png)

I have seen this with the reference [NodeMICMAC dataset](https://dronemapper.com/software/4thAve.zip), though to a lesser extent:
![image](https://user-images.githubusercontent.com/1174901/58280704-3ecd6200-7d6f-11e9-9840-c0068e0e0962.png)

This is a similar issue to what we saw in ODM here:
https://github.com/OpenDroneMap/ODM/issues/801

Fixed in mvs-texturing here: https://github.com/OpenDroneMap/mvs-texturing/pull/3 https://github.com/OpenDroneMap/mvs-texturing/pull/3/commits/7c01bca811467679f535ef292f6393e256e1acec

**Comment by dronemapper-io** [comment]
Thanks. Yes, currently there isn't a seamline feathering option in stock MicMac so that radiometric balance routine tries its best to blend all tiles. It is more apparently when you have camera or lighting issues. White balance on 'Auto` really causes problems. Definitely something to improve on. JP

**Comment by smathermather** [comment]
Ah, makes sense. That also explains the issues over low-texture areas like the water, which I wasn't sure whether to include with this issue.

**Comment by dronemapper-io** [comment]
Implementing a seamline feather routine would solve the issue. There are some additional tweaks to the `Tawny` command and radiometric balancing that can also improve results.

**Comment by pierotofy** [comment]
There's an interesting overview of this in the MicMac manual (page 46):

![image](https://user-images.githubusercontent.com/1951843/58284834-83f69180-7d79-11e9-8792-ef6ecddb237d.png)

I wonder what happens if you pass `DEq=1 De-gRapXY=[2,0]` to Tawny.

https://raw.githubusercontent.com/micmacIGN/Documentation/master/DocMicMac.pdf

**Comment by dronemapper-io** [comment]
Correct. Yes additional `Tawny` parameters can improve the results. One can also create a custom .xml and feed this into `Porto` with a larger blending window / etc.

**Comment by kikislater** [comment]
I got better results with this in Malt : 
`DefCor=0.0`

Then this in Tawny : 
`DEq=1 DegRapXY=[4,1] SzV=3 CorThr=0.6 NbPerIm=5e4`

**Comment by dronemapper-io** [comment]
Try `DEq=1 DegRapXY=[4,4] SzV=75 NbPerIm=5e4`

DefCor=0 will force MICMAC to correlate every pixel, even if it is a hidden point. It is good in some cases. Thanks

**Comment by kikislater** [comment]
> Try `DEq=1 DegRapXY=[4,4] SzV=75 NbPerIm=5e4`
> 
> DefCor=0 will force MICMAC to correlate every pixel, even if it is a hidden point. It is good in some cases. Thanks

Nice feebback, it was missing on MicMac I think ! Never tried this level of size window ... May be it needs a test with screenshots

**Comment by dronemapper-io** [comment]
Usually the ortho blending gets the best results when you run MICMAC to the highest zoom level (aka native resolution) as the ortho tiles are draped onto the DEM via nadir index score. Also, if the original raw imagery is from a sub-par camera or has white balance issues -- results will be bad.

**Comment by Ciaran1981** [comment]
Hi Folks,

I am rather late to this conversation, but a while back I adapted the seam-line feathering to multi-band if anyone is interested in my own python micmac lib.
https://ciaran1981.github.io/pycmac/build/html/pycmac.html#dense_match.feather

Ossim seems to perform better at this task in my opinion however....
